src/main/java/com/github/havardh/javaflow/exceptions/ExitException.java

```diff
@@ -0,0 +1,14 @@
+package com.github.havardh.javaflow.exceptions;
+
+public class ExitException extends RuntimeException {
+  private final int code;
+
+  public ExitException(int code, Throwable cause) {
+    super(cause);
+    this.code = code;
```

Comment:
I might suggest that we make the code an `enum` which tells us what the actual cause was with individual exit codes. 

```
enum ErrorCode {
  TYPE_MAP_NOT_FOUND(1),
  COULD_NOT_PARSE_SOURCE_CODE(2),
  // ...

  private int code;
  public ErrorCode(int code) { this.code = code; }
  
}

```

Reply:
Good idea, fixed! 👍